app/controllers/profile_controller.rb

```diff
@@ -6,9 +6,9 @@ def edit
 
   def update
     if current_user.update(user_params)
-      redirect_to edit_profile_path, notice: 'Данные обновлены'
+      redirect_to edit_profile_path, notice: t('flash.data_updated')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.